source/_components/sensor.growatt.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: "Growatt sensor"
+description: "Instructions on how to integrate a Growatt solar panel plant into Home Assistant."
+date: 2019-01-30 15:04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: growatt.png
+ha_category: Energy
```

Comment:
```suggestion
ha_category: 
  - Energy
  - Sensor
```

Reply:
Seems like CI can't handle this. What is your suggestion for the failure?